magento/magento2
# Error message in product review form not being translated

### Description
Add a validator to validate the rating and add the message to the i18n file

### Fixed Issues (if relevant)
1. magento/magento2#10474: Error message in product review form not being translated

### Manual testing scenarios
1. Make sure the review module is enabled and reviews are enabled on the product page
2. Go to "Stores > Ratings" and make one rating is active and selected for the selected storeview in step 3
3. Configure a storeview with a different language than en_US
4. Try to submit an empty review on a product page

### Contribution checklist
